daml-script/daml/Daml/Script.daml

```diff
@@ -121,6 +124,24 @@ data CommandsF a
 newtype Commands a = Commands (Ap CommandsF a)
   deriving (Functor, Applicative)
 
+data DisclosedContract = DisclosedContract
+  with
+    contractId : AnyContractId
+    contractKey : Optional AnyContractKey
```

Comment:
This doesn’t work. You need to actually include the metadata here in the same form we do over the gRPC API. Otherwise, we cannot run Daml Script over gRPC. I believe we discussed before that for the binary blobs, we can just base64 encode them in a Text.

Reply:
I see that for the JSON API we wen't with base16. Should this be the same here?

Reply:
Sounds good :+1:

Reply:
Encoded in base16 now.